### ISSUE TITLE
Screen reader does not read linked ranges correctly

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#280613]
Testing #279734

Voiceover on macOS, I would expect the link to be read as `<name> line <a> to <b>`

<img width="621" height="122" alt="Image" src="https://github.com/user-attachments/assets/7f46726a-de37-4d06-8f2b-05d8589f506c" />

**Comment by mjbvz** [comment]
Also seems worth fixing in November if possible
